src/pyvesync/vesync.py

```diff
@@ -51,14 +51,17 @@
     'Core200S': VeSyncAir200S,
     'Core300S': VeSyncAir300S400S,
     'Core400S': VeSyncAir300S400S,
+    'LUH-D301S-WEU': VeSyncHumid200300S,
+    'LAP-C201S-AUSR': VeSyncAir200S
 }
 
 _DEVICE_TYPES_DICT: Dict[str, List[str]] = dict(
     outlets=['wifi-switch-1.3', 'ESW03-USA',
              'ESW01-EU', 'ESW15-USA', 'ESO15-TB'],
     switches=['ESWL01', 'ESWL03', 'ESWD16'],
     fans=['LV-PUR131S', 'Classic200S', 'Classic300S', 'Core200S',
-          'Core300S', 'Core400S', 'Dual200S'],
+          'Core300S', 'Core400S', 'Dual200S',
+          'LUH-D301S-WEU', 'LUH-D301S-WEU'],
```

Comment:
Copy/Paste issue here? Missing the LAP-C201S-AUSR and duplicated the LUH-D301S-WEU device.

Reply:
Thank you for catching that!